src/components/BlockchainList/BlockchainPost.tsx

```diff
@@ -68,7 +71,7 @@ export default function ({
   const { ref, inView } = useInView()
 
   return (
-    <div ref={ref}>
+    <div ref={ref} style={inView ? {} : { height: '150px' }}>
```

Comment:
Maybe we can have
```TS
style = {{height: inView ? 'fit' : '150px'}}
```

Reply:
We may. Changed it to `fit-content` instead. Thx